### DISCUSSION TITLE
Version 1.24.0

### DISCUSSION THREAD

**Discussion by MM2-0** [MM2-0/Kvaesitso#343] [Announcements]
- Redesigned widget picker
- Added support for work profile widgets
- Restructured settings
- Widgets settings are now configured and stored per widget (not globally, as before). You may need to reapply them after the upgrade: press edit widgets and select the tune icon on the widget
- Added an option to remove the padding around app widgets
- You can now manually select which apps should appear in the music widget
- The icon picker has been improved: it now longer shows duplicate icons and for icon packs that support themed icons, both the themed and the unthemed variant are now included
- Bugfixes
- Disabled OneDrive integration (it was broken anyway and is likely to be removed in the future)

<hr /><em>This discussion was created from the release <a href='https://github.com/MM2-0/Kvaesitso/releases/tag/v1.24.0'>Version 1.24.0</a>.</em>

**Comment by shtrophic** [comment]
Great release! Swiping between the default horizontal pager layout is a lot smoother now :)
